component_library/scripts/build_scenario.py

```diff
@@ -331,57 +370,68 @@ def add_sequences(self, custom_timeseries=None):
                                     f"Column '{col_name}' missing from resource '{res.name}' although it is listed as foreignKey")
             # WIP, here need to take an external file as argument and only select 'profiles_to_add' columns
 
-            df_profiles = []
+            # df_profiles = []
+            # TODO: AVAILABLE_SEQUENCES seemed to be empty so this part is not used -> work around below
```

Comment:
AVAILABLE_SEQUENCES should not be empty, if it is, something is not correct in your local setting.

Reply:
You can run `python utils.py` (in the same folder as `utils.py`)  and it will show you if there are errors in the reading of the component of the WIP component library